stories/topics/proc-gcp-deleting-backups-playbook.adoc

```diff
@@ -50,21 +31,23 @@ gcp_backups_delete:
     gcp_bucket_backup_name:
 ----
 
-With backup_names like ["backup1","backup2"]
+Parameter *backup_names* MUST specify an array of strings.  For example: ["backup1","backup2"].
+Parameter *delete* MUST specify *true* to verify the deletion should occur.  Otherwise the delete will be a dry run
```

Comment:
The delete (in backticks) parameter must be set to `true` to successfully delete.

Where does this parameter come from?

Reply:
Fixed `delete`